gateleen-cache/src/test/java/org/swisspush/gateleen/cache/storage/RedisCacheStorageTest.java

```diff
@@ -154,6 +154,9 @@ public void testCacheEntriesCount(TestContext context) {
 
         // prepare
         jedis.sadd(CACHED_REQUESTS, "cache_item_1", "cache_item_2", "cache_item_3");
+        jedis.set(CACHE_PREFIX + "cache_item_1", jsonObjectStr("payload_1"));
```

Comment:
@mcweba kindly review if this makes sense. I think the sequence of the test execution is random in here.